agent/mitm-socket/go/generate_cert.go

```diff
@@ -80,11 +80,11 @@ func NewAuthority() (*x509.Certificate, *rsa.PrivateKey, error) {
 	certFromDisk, err := readCertFromDisk(caFile)
 
 	if err != nil && !errors.Is(err, os.ErrNotExist) {
-		log.Printf("Error reading cert from disk", caFile, err)
+		log.Print("Error reading cert from disk", caFile, err)
```

Comment:
Did gofmt automatically change these? Or is there a reason to change them to Print?

Reply:
You should get the warning with std go tooling, even just your compiler.
Anyway, question is more, why Printf in the first place, you're not formatting any args :P

Reply:
Ah, good point.. ha